### ISSUE TITLE
[Accessibility Bug] Review usage of aria-label

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#2633]
[This is a ticket for the "accessibility backlog"]

* Hamburger Menu button has too many `aria-label` values related to it
    * `aria-label` attribute on the button although it has got text content inside of it. And in the German translation visible text (Men**ü**) and `aria-label` (Menu) differ, which is a violation of [WCAG success criterion 2.5.3 "Label in name"](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html) and will cause problems in Voice Recognition software
    * The "three stripes" icon has itself an `aria-label` attribute of icon (!). This icon is purely decorative, influences the accessible name of the button in some user agents, and should receive `aria-hidden="true"`
* The same (icons purely decorative) is valid for the icons 
    * in the main menu
    * in the breadcrumb navigation
    * in the New File Menu

**Comment by pascalwengerter** [comment]
Closing this since #5018 has been merged
